pyproject.toml

```
@@ -18,7 +18,8 @@ classifiers = [
 ]
 requires-python = ">=3.8"
 dependencies = [
-	"zipp>=0.5",
+	"zipp>=0.5"; python_version < "3.8"',
+	"zipp>=3.19.1"; python_version >= "3.8"',
```

Since this library requires Python 3.8+, we can simply declare the dependency once:

```suggestion
	"zipp>=3.19.1",
```

That also fixes the syntax error due to the extra quote character.